CODE_STYLE.md

```diff
@@ -23,3 +23,6 @@ function test(uint256 _testParameter1, uint256 _testParameter2) {
     ...
 }
 ```
+
+* Events should be emitted immediately after the state change that they
+  represent, and consequently they should be named in past tense.
```

Comment:
Should we provide a couple examples? I'd add a note mentioning that some standards (e.g. ERC20) use present tense, and in those cases the spec prevails.

Reply:
Good ideas @nventuro! Please check if you like it.

Reply:
Great!